addons/hr_calendar/models/calendar_event.py

```diff
@@ -24,7 +24,7 @@ def _compute_unavailable_partner_ids(self):
         # based on company calendar's interval.
         for event, event_interval in event_intervals.items():
             start = event_interval._items[0][0]
-            stop = event_interval._items[0][1]
+            stop = event_interval._items[-1][1]
```

Comment:
This doesn't work if `_items` is empty -> current error in runbot.

Reply:
Yep I will add a check :D

Reply:
Well, with all the details we discussed in DM I guess the patch we discussed wouldn't need the check ... but it doesn't hurt :)

Reply:
> I don't know If you see it but I made a commit for your issue ;)

No, I didn't. At first sight it looks good.